public/content/documentation/web/component/link.md

```diff
@@ -6,75 +6,485 @@ How to test a link
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to focus on a link
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a link
+   - THEN when I doubletap with the link in focus I HEAR my browser goes somewhere
 
-   - I HEAR It typically contains the name and primary navigation of the website
 
-3. Mobile screenreader
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
 
-   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
 
-   - I swipe to focusable elements in the header
+## Links vs buttons
 
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+### If it goes somewhere, it's `<a>` link.
 
-   - I HEAR It typically contains the name and primary navigation of the website
+- When the user clicks a link, they are taken to a different location in the site.
+  - Either another page or even another area of the same page
+- A link can look like a big shiny button but it must be coded as `<a>` link
 
+### If it does something, it's a `<button>`
 
-Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
+- Buttons cause an action to occur on the same page
+  - Submit a form (even when submission takes you to a new page)
+  - Open a menu
+  - Launch a modal
+  - Expand details
+- A button can look like a link, but it must be coded as a `<button>`
+
+## Code examples
+
+### Use semantic HTML with common sense names
+
+This semantic HTML contains all accessibility features by default. 
+
+```html
+<a href="/about/">
+  About
+</a>
+```
+
+<example>
+<a href="/about/">
+  About
+</a>
+</example>
+
+### Name links logically
+
+- **Do not** use a heading with a generic link below. 
+- Instead, make the heading a link or programmatically associate the link with the heading using <code>aria-describedby</code>.
+
+#### Bad example
+
+```html
+<h3>About our coffee subscriptions</h3>
+<p>Get the best coffee delivered to your door</p>
+<a href="/about/">
+   Learn more
+</div>
+```
 
-## Developer Notes
+#### Good examples
 
-### Name
+```html
+<h3><a href="/about/">About our coffee subscriptions</a></h3>
+<p>Get the best coffee delivered to your door</p>
+```
 
-- Typically doesn’t have a name or description since there must be only one instance per page.
+```html
+<h3 id="unique-id">About our coffee subscriptions</h3>
+<p>Get the best coffee delivered to your door</p>
+<a href="/about/" aria-describedby="unique-id">
+   Learn more
+</div>
+```
 
-## Videos
+### Making a link with no `href` focusable
 
-- Videos go here
-<video controls>
-  <source src="media/video/native/button/buttonIosVoiceover.webm" type="video/webm">
-  Your browser does not support the video tag.
-</video>
+- **Do not** put anything but a URI in the `href`
```

Comment:
I came across this in another component page, so it is possible we'll come across it again - but this should be URL, not URI.